### PR TITLE
Custom SDL Logging Instructions

### LINKED ISSUE #76: Add guide for using log macros in Swift
Add more details for how to use the SDL log macros in a Swift project.
1. If using Cocoapods, the subspec must be installed using pod `SmartDeviceLink/Swift`
1. The `SmartDeviceLinkSwift` target must be imported before `SDLogD`, `SDLLogE`, etc. can be used.

### PR DESCRIPTION
**Pull request by NicoleYarroch**
Fixes #76

This pull request **adds new content**.

## Summary of Changes
Added section to **Developer Tools > Configuring SDL Logging** with instructions on how to create custom SDL logs in both Swift and Objective-C.

### CODE REVIEW

File: docs/Developer Tools/Configuring SDL Logging/index.md [hunk 1]
```diff
@@ -94,3 +102,39 @@ SDLLogFilter *filter = [SDLLogFilter filterByDisallowingString:@"Test" caseSensi
 ```swift
 let filter = SDLLogFilter(disallowingString: "Test", caseSensitive: false)
 ```
+
+### Add Custom SDL Logs to Your Project
```
Comment (joeljfischer) [inline docs/Developer Tools/Configuring SDL Logging/index.md]: I'm not sure what "custom" means here. I think `### Logging with the SDL Logger` may be clearer?

File: docs/Developer Tools/Configuring SDL Logging/index.md [hunk 2]
```diff
@@ -94,3 +102,39 @@ SDLLogFilter *filter = [SDLLogFilter filterByDisallowingString:@"Test" caseSensi
...
+In addition to viewing the library logs, you also have the ability to add custom SDL logs to your project. The behavior of custom SDL logs, just like the library SDL logs, is based on your `SDLLogConfiguration` settings.
```
Comment (joeljfischer) [inline docs/Developer Tools/Configuring SDL Logging/index.md]: > you also have the ability to add custom SDL logs to your project

you also have the ability to log with the SDL logger.

> The behavior of custom SDL logs, just like the library SDL logs, is based on your `SDLLogConfiguration` settings.

All messages logged through the SDL logger, including your own, will use your `SDLLogConfiguration` settings.

File: docs/Developer Tools/Configuring SDL Logging/index.md [hunk 3]
```diff
@@ -94,3 +102,39 @@ SDLLogFilter *filter = [SDLLogFilter filterByDisallowingString:@"Test" caseSensi
...
+
+#### Objective-C Projects
+Simply use the convenient log macros to create a custom SDL log in your project.
```
Comment (joeljfischer) [inline docs/Developer Tools/Configuring SDL Logging/index.md]: Mention that you must `#import "SDLLogMacros.h"`?

File: docs/Developer Tools/Configuring SDL Logging/index.md (hunk 1 above)
Comment (NicoleYarroch) [reply docs/Developer Tools/Configuring SDL Logging/index.md]: Fixed

File: docs/Developer Tools/Configuring SDL Logging/index.md (hunk 2 above)
Comment (NicoleYarroch) [reply docs/Developer Tools/Configuring SDL Logging/index.md]: Fixed

File: docs/Developer Tools/Configuring SDL Logging/index.md (hunk 3 above)
Comment (NicoleYarroch) [reply docs/Developer Tools/Configuring SDL Logging/index.md]: Added.

**Pull request merged**